### DISCUSSION TITLE
how to delete node after the node already doesn't in the cluster

### DISCUSSION THREAD

**Discussion by wxcoin** [apache/iotdb#14372] [General]
IoTDB> show cluster
+------+----------+-------+---------------+------------+-------+---------+
|NodeID|  NodeType| Status|InternalAddress|InternalPort|Version|BuildInfo|
+------+----------+-------+---------------+------------+-------+---------+
|     0|ConfigNode|Running|        iotdb-1|       10710|  1.3.0|  d1326c5|
|     4|ConfigNode|Unknown|        iotdb-2|       10710|  1.3.0|  d1326c5|
|     5|ConfigNode|Running|        iotdb-3|       10712|  1.3.0|  d1326c5|
|     1|  DataNode|Running|        iotdb-1|       10730|  1.3.0|  d1326c5|
|     3|  DataNode|Unknown|        iotdb-2|       10730|  1.3.0|  d1326c5|
|     6|  DataNode|Running|        iotdb-3|       10732|  1.3.0|  d1326c5|
+------+----------+-------+---------------+------------+-------+---------+

**Comment by 2b3c511** [comment]
You can execute ./sbin/remove-confidnode.sh or ./sbin/remove-datanode.sh script specifies the node ID parameter to remove nodes. However, before removal, it is necessary to ensure that the number of available nodes in the cluster is greater than max (the number of metadata replicas, the number of data replicas) in order to successfully remove.
